library/src/main/java/com/billing/BillingServiceListener.java

```diff
@@ -10,4 +10,9 @@
      * @param iapKeyPrices - a map with available products
      */
     void onPricesUpdated(Map<String, String> iapKeyPrices);
+
+    /**
+     * Callback will be triggered upon the inventory query is done
+     */
+    void onInventoryQueryFinished();
```

Comment:
There is a thought that this method is very specific to the app needs and will be used only in some rare cases. 
Do we need to force all IAP lib users to integrate this new method on their projects? Can we do it optional?


As the solution we can migrate listeners from Java to Koltin and make 'onInventoryQueryFinished' to be with default implementation. Something like this
```
    /**
     * Callback will be triggered upon the inventory query is done
     */
    fun onInventoryQueryFinished(){}

```
This will allow the developer to override this method only when he really needs it. Otherwise, the compiler will skip it.

Reply:
Hey guys, it's up to you wether you want to change code style, and make it optional or not, but let me explain my use case and why I think this is not a very specific use case, this is only not your use case:
For example I need it for a purchase that can make a free app version become a pro version.
Many things in the app depend on this (wether it is free or pro) so the purchase check needs to be done on application startup, and this check must be sure that the purchase has been properly queried (otherwise there is a chance the app could start with free flag while it should be pro).
Do you see more how such a callback can be used ?
Cheers !